docs/OMS-Agent-for-Linux.md

```diff
@@ -361,6 +361,7 @@ If you are using selinux, the semanage tool can be used to allow TCP traffic for
 	  tag oms.syslog
 </source>
 ```
+**Note:** ports 0-1024 are privileged, and require special permissions. As the OMS Agent for Linux runs as the `omsagent` user trying to allocate a source port of 0 to 1024 results in agent startup failure
```

Comment:
I'm not sure this is the best of English, but if it makes sense to you, I guess it's fine. I certainly understand what you mean, but I'm unsure if an unexperienced user would.

Perhaps reword to something like:

**Note:** Ports 1-1024 are privileged, and require root permissions. Since the OMS Agent runs as the `omsagent` user (not `root`), using a port between 1 thru 1024 will result in an agent startup failure.